docs/amber_script.md

```diff
@@ -90,11 +89,13 @@ BUFFER <buffer_type> <name> DATA_TYPE <type> DATA
 <value>+
 END
 
-BUFFER <buffer_type> <name> DATA_TYPE <type> SIZE <size_in_bytes> <initializer>
+BUFFER <buffer_type> <name> DATA_TYPE <type> SIZE <size_in_items> <initializer>
```

Comment:
What does `size_in_items` mean?

Reply:
It's the number of items in the buffer, as opposed to the size_in_bytes, so 5 int32's would have a size_in_items of 5 and a size_in_bytes of 20. I wanted to make it clear what number we expected here.